setup.go

```diff
@@ -25,48 +26,48 @@ var EXECUTION_DIR string
 const SUMMARY_TEST_DIR = "testdata/bbi_summary"
 const SUMMARY_GOLD_DIR = "aa_golden_files"
 const noSuchFileError = "no such file or directory"
-const noFilePresentError = "No file present at"
+const noFilePresentError = "no file present at"
```

Comment:
This was literally the only thing that needed to be changed after your conversion to wrapt